src/main/java/uk/gov/justice/hmpps/prison/api/model/RequestForNewBooking.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.validator.constraints.Length;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.validation.constraints.NotNull;
+import java.time.LocalDateTime;
+
+@ApiModel(description = "Represents the data required receiving a prisoner under a new booking")
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@ToString
+public class RequestForNewBooking {
+
+    @ApiModelProperty(value = "Received Prison ID", example = "MDI", position = 1)
+    @Length(max = 3, message = "Prison ID")
+    @NotNull
+    private String receivedPrisonId;
```

Comment:
Observation, it would be nice to have a single ubiquitous language - especially were we have agencyId, locationId, agency, prisonId all used. I suppose movements can be to non-prisons that confuse things, but I like `receivedPrisonId` or maybe just `prisonId`